dotcom-rendering/src/layouts/PictureLayout.tsx

```diff
@@ -0,0 +1,757 @@
+	<div
+		css={css`
+			/* IE Fallback */
+			display: flex;
+			flex-direction: column;
+			${until.leftCol} {
+				margin-left: 0px;
+			}
+			${from.leftCol} {
+				margin-left: 151px;
+			}
+			${from.wide} {
+				margin-left: 230px;
+			}
+
+			@supports (display: grid) {
+				display: grid;
+				width: 100%;
+				margin-left: 0;
+
+				grid-column-gap: 10px;
+
+				/*
+					Explanation of each unit of grid-template-columns
+
+					Left Column (220 - 1px border)
+					Vertical grey border
+					Main content
+					Right Column
+					
+				*/
+				${from.wide} {
+					grid-template-columns: 219px 1px 1fr;
+					grid-template-areas:
+						'title  border  headline'
+						'lines  border  media'
+						'meta   border  media'
+						'meta   border  standfirst'
+						'.      border  submeta'
+						'.      border  .';
+				}
+
+				${until.wide} {
+					grid-template-columns: 140px 1px 1fr 300px;
+					grid-template-areas:
+						'title  border  headline    headline'
+						'lines  border  media       media'
+						'meta   border  media       media'
+						'meta   border  standfirst  standfirst'
+						'.      border  submeta     submeta'
+						'.      border  .           . ';
+				}
+
+				/*
+					Explanation of each unit of grid-template-columns
+
+					Main content
+					Right Column
+				*/
+				${until.leftCol} {
+					grid-column-gap: 0px;
+					grid-template-columns: 1fr; /* Main content */
+					grid-template-areas:
+						'title     '
+						'headline  '
+						'lines     '
+						'meta      '
+						'media     '
+						'standfirst'
+						'submeta   '
+						'.         ';
+				}
+
+				${until.desktop} {
+					grid-column-gap: 0px;
+					grid-template-columns: 1fr; /* Main content */
+					grid-template-areas:
+						'title'
+						'headline'
+						'lines'
+						'meta'
+						'media'
+						'standfirst'
+						'submeta';
+				}
+
+				${until.tablet} {
+					grid-column-gap: 0px;
+					grid-template-columns: 1fr; /* Main content */
+					grid-template-areas:
+						'title'
+						'headline'
+						'lines'
+						'meta'
+						'media'
+						'standfirst'
+						'submeta';
+				}
+			}
+		`}
+	>
+		{children}
+	</div>
+);
+
+const maxWidth = css`
+	${from.desktop} {
+		max-width: 620px;
+	}
+`;
+
+const mainMediaWrapper = (displayAvatarUrl: boolean) => css`
+	position: relative;
+	${until.phablet} {
+		margin-left: 20px;
+		margin-right: 20px;
+	}
+	${until.mobileLandscape} {
+		margin-left: 10px;
+		margin-right: 10px;
+	}
+	${displayAvatarUrl
+		? css`
+				margin-top: 8px;
+		  `
+		: ``}
+`;
+
+const avatarHeadlineWrapper = css`
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
+`;
+
+// If in mobile increase the margin top and margin right deficit
+const avatarPositionStyles = css`
+	display: flex;
+	justify-content: flex-end;
+	position: relative;
+	margin-bottom: -29px;
+	margin-top: -50px;
+	pointer-events: none;
+	${until.tablet} {
+		overflow: hidden;
+	}
+
+	/*  Why target img element?
```

Comment:
This is interesting. Another way would be to send a prop down to the `ContributorAvatar`. I agree with the comment though that that would be splitting the implementation into bits that coupled / the same thing.

Not being to close to the issue, the question I might ask is "Do we need to have a different avatar on pictures than we have elsewhere?"

Reply:
Thinking more deeply about that question, I suppose what we might want to remember is when we get to audio / video etc, should we be trying to get them to fit into this Layout instead of proliferating even further on the layouts for small, potentially insignificant changes.

(To be clear, I really believe splitting this out into it's own layout is the right idea, and hope we can use it elsewhere).

Reply:
Could have made this more explicit, this bit of logic is [taken from](https://github.com/guardian/dotcom-rendering/blob/87a6b908d2a3264cabb709c1334494078906ef13/dotcom-rendering/src/layouts/CommentLayout.tsx#L212) `CommentLayout.tsx`, as we want the same outcome.

Looking at this more closely - `Avatar.tsx` gets used in most places, but `ContributorAvatar.tsx` so far only seems to be used in the comments section headline, which makes me wonder if this related `avatarPositionStyles` css could be included directly in `ContributorAvatar.tsx` (especially as it's now being repeated in `PictureLayout.tsx`). But also, from a quick glance, it looks like those avatar components differ mainly in image quality and some styling (notably width/height), so maybe there's an opportunity to refactor and simplify the avatar logic overall?